### ISSUE TITLE
Incorrect ADF paths in ini files 

### ISSUE THREAD

**Issue by darius600** [dirkwhoffmann/vAmigaTS#5]
First of all, thank you very much for this impressive collection of test cases. To me, this looks like the most complete and systematic way of testing any Amiga recreation. I greatly appreciate the effort that has gone into this project!

I believe I have stumbled upon an issue related to the paths in the .ini files:
Around 1000 .ini files contain paths to ADF files that include either `_ecs.adf` or `_ocs.adf`. 
However, these ADF files do not exist because the actual ADF file names do not contain any chipset designation (i.e., are missing the `_ecs` or `_ocs` suffix).

**Comment by dirkwhoffmann** [comment]
tl;dr: This is actually a feature to not blow up the repo too much. During regression testing, the file gets renamed on-the-fly to `/tmp/<base name of the ini-file>.adf`.

Detailed explanation: The main purpose of vAmigTS was to provide an automatic regression test environment for vAmiga, which means that the actual tests are carried out by executing a bunch of Makefiles. 

For an example, consider test case ddf1:

```console
ls Agnus/DDF/DDF/ddf1
ddf1.adf		ddf1_ecs.ini		ddf1_ocs.raw
ddf1.s			ddf1_ecs.raw		ddf1_ocs.tiff
ddf1_A500_ECS.JPG	ddf1_ecs.tiff
ddf1_A500_OCS.JPG	ddf1_ocs.ini
```

The directory contains a single test-case, thus a single ADF. However, it is supposed to run with different machine configs, thus multiple `.ini` files exist. 

Execution of this test is done by a Makefile that links to `Scripts/Makefile3`. Besides others, this file contains the following line, showing what's going on:

```Makefile
%.log: %.tiff %.raw
	@cp $(wildcard $(dir $@)*.adf) /tmp/$(basename $(@F)).adf
	@$(VAMIGA) $(basename $@).ini &> /dev/null
	@diff /tmp/$(basename $(@F)).raw $(basename $@).raw; \
```

After scanning the directory for `.ini` files, it runs the emulator for each of these files. In each run, it takes whatever ADF it finds and renames it to fit the base name of the `.ini` file. The renamed file is copied to the `/tmp` directory and picked up by the script from there.

**Comment by darius600** [comment]
Thank you for the detailed explanation – I understand.
Just for curiosity: What is the reason for duplicating the ADF files? Is this that if multiple instances of the vAmiga run in parallel the file is not locked?

**Comment by dirkwhoffmann** [comment]
> What is the reason for duplicating the ADF files?

Yes. When performing a regression test, I usually run between 6 and 9 emulator instances in parallel (e.g. `make -j6`). This only works when all ADFs have a unique name. The ADFs are copied to `/tmp` before running the test because it made the regression tester easier to implement. I don't need to pass a search path to vAmiga. It knows that they can be picked up in `/tmp`.
